iothub/service/tests/IotHubServiceClientTests.cs

```diff
@@ -0,0 +1,88 @@
+            using var serviceClient = new IotHubServiceClient(cs);
+
+            // assert
+            serviceClient.Devices.Should().NotBeNull();
+            serviceClient.Modules.Should().NotBeNull();
+            serviceClient.Configurations.Should().NotBeNull();
+            serviceClient.DirectMethods.Should().NotBeNull();
+            serviceClient.Query.Should().NotBeNull();
+            serviceClient.ScheduledJobs.Should().NotBeNull();
+            serviceClient.DigitalTwins.Should().NotBeNull();
+            serviceClient.Twins.Should().NotBeNull();
+            serviceClient.MessageFeedback.Should().NotBeNull();
+            serviceClient.FileUploadNotifications.Should().NotBeNull();
+            serviceClient.Messages.Should().NotBeNull();
+        }
+
+        [TestMethod]
+        public void IotHubServiceClient_CreateWithAAD()
+        {
+            // arrange
+            string hostName = "acme.azure-devices.net";
+            var tokenCrediential = new TestTokenCredential(new DateTimeOffset(DateTime.Now + TimeSpan.FromHours(1)));
+
+            // act
+            using var serviceClient = new IotHubServiceClient(hostName, tokenCrediential);
+
+            // assert
+            serviceClient.Devices.Should().NotBeNull();
+            serviceClient.Modules.Should().NotBeNull();
+            serviceClient.Configurations.Should().NotBeNull();
+            serviceClient.DirectMethods.Should().NotBeNull();
+            serviceClient.Query.Should().NotBeNull();
+            serviceClient.ScheduledJobs.Should().NotBeNull();
+            serviceClient.DigitalTwins.Should().NotBeNull();
+            serviceClient.Twins.Should().NotBeNull();
+            serviceClient.MessageFeedback.Should().NotBeNull();
+            serviceClient.FileUploadNotifications.Should().NotBeNull();
+            serviceClient.Messages.Should().NotBeNull();
+        }
+
+        [TestMethod]
+        public void IotHubServiceClient_CreateWithSasToken()
+        {
+            // arrange
+            string hostName = "acme.azure-devices.net";
+            var sasCredential = new AzureSasCredential("test");
+
+            // act
+            using var serviceClient = new IotHubServiceClient(hostName, sasCredential);
+
+            // assert
+            serviceClient.Devices.Should().NotBeNull();
+            serviceClient.Modules.Should().NotBeNull();
+            serviceClient.Configurations.Should().NotBeNull();
+            serviceClient.DirectMethods.Should().NotBeNull();
+            serviceClient.Query.Should().NotBeNull();
+            serviceClient.ScheduledJobs.Should().NotBeNull();
+            serviceClient.DigitalTwins.Should().NotBeNull();
+            serviceClient.Twins.Should().NotBeNull();
+            serviceClient.MessageFeedback.Should().NotBeNull();
+            serviceClient.FileUploadNotifications.Should().NotBeNull();
+            serviceClient.Messages.Should().NotBeNull();
```

Comment:
Could we have unit test cases for https://github.com/Azure/azure-iot-sdk-csharp/blob/previews/v2/iothub/service/src/IotHubServiceClient.cs#L84-L85?